src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -74,27 +76,32 @@ const handleClick = (): void => {
 
 <style lang="scss" scoped>
 @import '@/styles/variables';
+@import '@/styles/tmp-variables';
 @import '@/styles/functions';
 
 .k-multiselect-item {
-  margin-bottom: 4px;
+  margin-bottom: var(--kui-space-20, $kui-space-20);
 
   &:last-of-type {
-    margin-bottom: 0 !important;
+    margin-bottom:var(--kui-space-0, $kui-space-0) !important;
+  }
+
+  .k-multiselect-item-container {
+    display: block !important
   }
 
   button {
     align-items: center;
-    background-color: var(--white);
+    background-color: var(--white, var(--kui-color-background, $kui-color-background));
     border: none;
-    border-radius: 4px;
-    color: var(--grey-500);
+    border-radius: var(--kui-border-radius-20, $kui-border-radius-20);
+    color: var(--grey-500, var(--kui-color-text-neutral, $kui-color-text-neutral));
     display: flex;
-    font-family: var(--font-family-sans);
-    font-size: var(--type-xs);
-    font-weight: 400;
-    line-height: 26px;
-    padding-left: var(--spacing-xxs);
+    font-family: var(--font-family-sans, var(--kui-font-family-text, $kui-font-family-text));
+    font-size: var(--type-xs, var(--kui-font-size-20, $kui-font-size-20));
+    font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
+    line-height: var(--kui-line-height-40, $kui-line-height-40);
```

Comment:
question: shouldn't the line-height match the font size token?

Reply:
No necessarily - sometime we might wanna give it some extra room maybe?

Reply:
That doesn't match what Jason told us. Can you use the corresponding tokens?

Reply:
Done